script.js

```diff
@@ -0,0 +1,62 @@
+let bookList = JSON.parse(localStorage.getItem('books')) || [];
+
+function addBook(title, author) {
```

Comment:
- [ ] OPTIONAL
The book _title_ and book _author_ input are both accepting numbers as inputs. Kindly endeavor to provide validation for the same